### ISSUE TITLE
[FR]:add the yandex music service, there is an open api

### ISSUE THREAD

**Issue by saber716rus** [mixxxdj/mixxx#11958]
### Feature Description

there is an api posted on github. https://github.com/MarshalX/yandex-music-api

**Comment by ronso0** [comment]
What exactly would be the outcome, who would a Mixxx integration look like?

**Comment by saber716rus** [comment]
How deezer and soundcloud services were integrated. That is, the ability to download tracks to the soundboard from this service.

**Comment by ronso0** [comment]
> How deezer and soundcloud services were integrated

Integrated where?
Certainly not in Mixxx since the Temrs & Conditions (of Soundcloud at least) do not allow downloading the music files as long as the app using them is able to record the audio output (mixed or not).

This has been discussed at length. For reference:
https://github.com/mixxxdj/mixxx/issues/6277
https://mixxx.zulipchat.com/#narrow/stream/109122-general/topic/Soundcloud.20integration

**Comment by saber716rus** [comment]
Yes, you understood me correctly. So, I understand that with the license of these services, too, not everything is so smooth. I asked them again about licensing.
